framework/src/main/java/org/checkerframework/framework/type/treeannotator/PropagationTreeAnnotator.java

```diff
@@ -41,9 +49,17 @@
  */
 public class PropagationTreeAnnotator extends TreeAnnotator {
 
+  /** The qualifier hierarchy to use. */
   private final QualifierHierarchy qualHierarchy;
 
-  /** Creates a {@link PropagationTreeAnnotator} for the given {@code atypeFactory}. */
+  /** Used to replace annotations in var type according to initializer annotations. */
+  private final AnnotatedTypeReplacer replacer = new AnnotatedTypeReplacer();
```

Comment:
There's no need to create a new replacer.  Call `atypeFactory.replaceAnnotations(initializerType, variableType);`
 instead.